src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -93,6 +93,49 @@ public class MatchInfo
         /// <value>The text of the matching line.</value>
         public string Line { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the matched portion of the string is highlighted.
+        /// </summary>
+        /// <value>Whether the matched portion of the string is highlighted with asterisks.</value>
+        public bool Emphasize { get; set; }
+
+        /// <summary>
+        /// Stores the starting index of each match within the line.
+        /// </summary>
+        private readonly List<int> _matchIndexes;
+
+        /// <summary>
+        /// Stores the length of each match within the line.
+        /// </summary>
+        private readonly List<int> _matchLengths;
+
+        /// <summary>
+        /// Stores a values indicating whether or not the terminal supports VT.
+        /// </summary>
+        private readonly bool _supportsVirtualTerminal;
```

Comment:
I believe we should remove this from MatchInfo. This is only used in `ToEmphasizedString()`
where we can directly use `Host.UI.SupportsVirtualTerminal`.

Reply:
I don't think I can remove this easily. MatchInfo doesn't have access to `Host`, while SelectStringCommand does since it inherits from PSCmdlet.

Reply:
I moved this check into SelectStringCommand's doMatchWorker.